tasks/cycles/cycles.js

```diff
@@ -42,6 +89,15 @@ function calculateDiscount(redemption) {
     interestOnDeposit: 250,
     otherExpences: -300
   };
+
+  let a = 0;
+
+  for (var t in personIncomes){
+    a = a + personIncomes[t];
```

Comment:
similar to the above:
let sum = 0;
for (let x in personIncomes){
sum = sum + personIncomes[x]; OR sum += personIncomes[x]
}
console.log(sum)